docs/src/main/sphinx/release/release-353.md

```diff
@@ -0,0 +1,95 @@
+* Fix listing of schema registry tables having ambiguous subject name in lower case.
+
+## MySQL connector
+
+* Fix failure when reading a `datetime` value with more than 3 decimal digits of the second fraction. ({issue}`6852`)
+* Fix incorrect predicate pushdown for `char` and `varchar` column with operators like `!=`, `<`, `<=`, `>` and `>=` due
+  different case sensitivity between Trino and MySQL. ({issue}`6753`)
+
+## MemSQL connector
+
+* Fix failure when reading a `datetime` value with more than 3 decimal digits of the second fraction. ({issue}`6852`)
+* Fix incorrect predicate pushdown for `char` and `varchar` column with operators like `!=`, `<`, `<=`, `>` and `>=` due
+  different case sensitivity between Trino and MemSQL. ({issue}`6753`)
+
+## PostgreSQL connector
+
+* Improve performance of queries with `ORDER BY ... LIMIT` clause, when the computation can be pushed down to the underlying database.
+  This can be enabled by setting `topn-pushdown.enabled`. Enabling this feature can currently result in incorrect query results when sorting
+  on `char` or `varchar` columns. ({issue}`6847`)
+
+## Phoenix connector
+
+* Add support for Phoenix 5.1. To use connect to Phoenix 5.1 server user `connector.name=phoenix-5` in catalog
+  configuration file. ({issue}`6865`)
+
+## Redshift connector
+
+* Fix failure when reading a `timestamp` value with more than 3 decimal digits of the second fraction. ({issue}`6893`)
+
+## SQL Server connector
+
+* Fix incorrect predicate pushdown for `char` and `varchar` column with operators like `!=`, `<`, `<=`, `>` and `>=` due
+  different case sensitivity between Trino and SQL Server. ({issue}`6753`)
+
+## SPI
+
+* Fix registering of lazy block listeners when top level block is already loaded.
+  Previously such listeners were never called when nested blocks were loaded. ({issue}`6783`)
+* Make `LazyBlock#getFullyLoadedBlock` load nested lazy blocks when top level block is already loaded.
+  Previously nested lazy blocks were not loaded in such case. ({issue}`6783`)
+* Fix `io.trino.connector.ConnectorAwareNodeManager#getWorkerNodes` including
+  coordinator when `node-scheduler.include-coordinator` is disabled. ({issue}`7007`)
+* The function name passed to `io.trino.spi.connector.ConnectorMetadata#applyAggregation` is now the canonical function name.
+  Previously, if query used function alias, the alias name was passed. ({issue}`6189`)
+* Extend redirection SPI with support for redirections to
+  multiple tables which are unioned together. ({issue}`6679`)
+* Change return type of `io.trino.spi.predicate.Range.intersect(Range)`. The method now returns `Optional.empty()`
+  instead of throwing when ranges do not overlap. ({issue}`6976`)
+* Change signature of `io.trino.spi.connector.ConnectorMetadata.applyJoin`. Method now takes additional `JoinStatistics` argument. ({issue}`7000`)
+* Deprecate `io.trino.spi.predicate.Marker`.
```

Comment:
This actually happened in 352 but i forgot a release note about this.

Reply:
We can leave it here for visibility and consider it deprecated as of this release, even though the class had the `@Deprecated` annotation in the previous version.